### PR TITLE
Add PYPY_VERSION file to the images

### PR DESCRIPTION
**Pull request by dmitry-shibanov**
# Description
In scope of this pull request we add new `PYPY_VERSION` file for all images. File will contain the version of PyPy. This changes are related to pull request with adding support for `setup-python` to install pypy.

## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by AlenaSviridenko — APPROVED** [review]

File: images/linux/scripts/installers/pypy.sh [hunk 1]
```diff
@@ -52,6 +52,11 @@ function InstallPyPy
 
     echo "Create additional symlinks (Required for UsePythonVersion Azure DevOps task)"
     cd $PYPY_TOOLCACHE_VERSION_ARCH_PATH/bin
+
+    PYPY_FULL_VERSION=$(./$PYPY_MAJOR -c "import sys;print('{}.{}.{}'.format(*sys.pypy_version_info[0:3]))")
```
Comment (maxim-lobanov) [inline images/linux/scripts/installers/pypy.sh:56]: I think it should be moved above where you get PYTHON_FULL_VERSION

File: images/linux/scripts/installers/pypy.sh [hunk 2]
```diff
@@ -52,6 +52,11 @@ function InstallPyPy
...
+    echo "PYPY_FULL_VERSION is $PYPY_FULL_VERSION"
```
Comment (maxim-lobanov) [inline images/linux/scripts/installers/pypy.sh:57]: nitpick comment: `echo "Put '$PYPY_FULL_VERSION' to PYPY_VERSION file"`

File: images/macos/provision/core/pypy.sh [hunk 3]
```diff
@@ -11,6 +11,8 @@ function InstallPyPy
     PACKAGE_URL=$1
 
     PACKAGE_TAR_NAME=$(echo $PACKAGE_URL | awk -F/ '{print $NF}')
+    versionV=$(echo $PACKAGE_URL | cut -f2 -d"-")
+    version=${versionV/"v"/""}
```
Comment (maxim-lobanov) [inline images/macos/provision/core/pypy.sh:15]: Why do we need these lines?

File: images/macos/provision/core/pypy.sh [hunk 4]
```diff
@@ -36,7 +38,6 @@ function InstallPyPy
     PYPY_TOOLCACHE_PATH=$AGENT_TOOLSDIRECTORY/PyPy
     PYPY_TOOLCACHE_VERSION_PATH=$PYPY_TOOLCACHE_PATH/$PYTHON_FULL_VERSION
     PYPY_TOOLCACHE_VERSION_ARCH_PATH=$PYPY_TOOLCACHE_VERSION_PATH/x64
```
Comment (maxim-lobanov) [inline images/macos/provision/core/pypy.sh:40]: nitpick: return line back :)

File: images/win/scripts/Installers/Install-PyPy.ps1 [hunk 5]
```diff
@@ -72,6 +72,10 @@ function Install-PyPy
         Write-Host "Symbolic link created for '$pypyArchPath\Scripts' <<===>> '$pypyArchPath\bin'"
         New-Item -Path "$pypyArchPath\bin" -ItemType SymbolicLink -Value "$pypyArchPath\Scripts" | Out-Null
 
+        $pypy_version = & $pypyApp -c "import sys;print('{}.{}.{}'.format(*sys.pypy_version_info[0:3]))"
```
Comment (maxim-lobanov) [inline images/win/scripts/Installers/Install-PyPy.ps1:75]: please use consistent naming

File: images/macos/provision/core/pypy.sh (hunk 4 above)
Comment (dmitry-shibanov) [reply images/macos/provision/core/pypy.sh]: Done.

File: images/win/scripts/Installers/Install-PyPy.ps1 (hunk 5 above)
Comment (dmitry-shibanov) [reply images/win/scripts/Installers/Install-PyPy.ps1]: Done.

File: images/macos/provision/core/pypy.sh (hunk 3 above)
Comment (dmitry-shibanov) [reply images/macos/provision/core/pypy.sh]: Done.

File: images/linux/scripts/installers/pypy.sh (hunk 2 above)
Comment (dmitry-shibanov) [reply images/linux/scripts/installers/pypy.sh]: Done.

File: images/linux/scripts/installers/pypy.sh (hunk 1 above)
Comment (dmitry-shibanov) [reply images/linux/scripts/installers/pypy.sh]: Done.

**Review by maxim-lobanov — APPROVED** [review]

**Review by al-cheb — APPROVED** [review]

**Pull request merged**